packages/string-templates/src/index.cjs

```diff
@@ -36,3 +36,8 @@ if (!process.env.NO_JS) {
     return vm.run(js)
   })
 }
+
```

Comment:
Seems like these files till have a dependency on `vm2` - is that correct?

Reply:
This is not actually used, I am working on removing it completely!